prov/efa/src/rdm/rxr_pkt_type_misc.c

```diff
@@ -454,8 +454,6 @@ void rxr_pkt_handle_rma_read_completion(struct rxr_ep *ep,
 			rxr_read_release_entry(ep, read_entry);
 		}
 	}
-
-	rxr_ep_record_tx_op_completed(ep, context_pkt_entry);
```

Comment:
Call `rxr_pkt_handle_rma_completion` here?

Reply:
This function is called by `rxr_pkt_handle_rma_completion`, so it cannot call the function

Reply:
I see. I misread the function name.

Could you check the CI failure. It seems real.